shared/reducers/plan-billing.js

```diff
@@ -10,48 +10,51 @@ const initialState: Constants.State = {
   usage: null,
 }
 
-export default function(state: Constants.State = initialState, action: Constants.Actions): Constants.State {
+export default function(
+  state: Constants.State = initialState,
+  action: Constants.Actions | {type: 'common:resetStore', payload: void}
+): Constants.State {
   switch (action.type) {
     case CommonConstants.resetStore:
       return {...initialState}
-    case Constants.updateBillingAndQuota:
-      if (action.error) {
-        console.warn('Error in action: ', action)
-        return state
-      }
-      return {
-        ...state,
-        ...action.payload,
-      }
-    case Constants.updateAvailablePlans:
-      if (action.error) {
-        console.warn('Error in action: ', action)
-        return state
-      }
-      return {
-        ...state,
-        ...action.payload,
-      }
-    case Constants.updatePaymentInfo:
-      if (action.error) {
-        console.warn('Error in action: ', action)
-        return state
-      }
-      return {
-        ...state,
-        ...action.payload,
-      }
-    case Constants.billingError:
-      if (action.error) {
-        return {
-          ...state,
-          errorMessage: action.payload.errorText,
-        }
-      }
-      return {
-        ...state,
-        errorMessage: null,
-      }
+    // case Constants.updateBillingAndQuota:
```

Comment:
turns out this is never used, leaving it commented for when we get to it later

Reply:
Unless we're gonna get to it really soon, I'd prefer that we delete code rather than comment it out (since it might rot). But I don't care that much about it.